i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx

```diff
@@ -22,6 +22,26 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
 
 ## Centreon Web
 
+### 23.04.14
+
+Release date: `January 12, 2024`
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [API] Fixed a visibility issue on configuration/users endpoint.
+- [Authentication] Fixed hardcoded email value that block SSO authentication with OpenId Connect when parameter is not email.
+- [Configuration] Fixed error message displayed in configuration generation after upgrading to 23.04 version.
+- [Event Logs] Fixed period in CSV export.
+- [Installation] Fixed issue of poller ID appearing twice in Centreon gorgone configuration.
+- [Installation] Fixed installation when database user already exists.
+- [My Account] Fixed account properties edition for local user only.
```

Comment:
```suggestion
- [My Account] Made the name, login and password fields read-only for non-local users so as to prevent issues where users who had edited these properties were locked out of the platform.
```